### ISSUE TITLE
Cone not working properly?

### ISSUE THREAD

**Issue by jzthree** [plotly/plotly.py#1051]
Thanks for implementing the feature. I have been having some difficulties in reproducing the notebook https://plot.ly/python/3d-cone/ or tried to use ploly.graph_objs.Cone.

With iplot it shows a plot with 2D layout like scatter while 3d is expected.
With plot to save to a file and open it with chrome. It shows an empty grid.

Also "Edit in Chart Studio" button does not work for me in either case.

I have verified that I am using 3.0.0 version, and the html files says plotly.js 1.38.3.   I have been mostly following the notebook except for using plotly.offline.plot and iplot instead.

**Comment by jzthree** [comment]
Thanks for looking into this! This is what i got 

![image](https://user-images.githubusercontent.com/8333155/42545769-f5c21d0c-8487-11e8-9e27-84d9e202303f.png)

```
from plotly.offline import  init_notebook_mode, plot, iplot
import plotly.graph_objs as go
init_notebook_mode()

data = [{
    'type': 'cone',
    'x': [1], 'y': [1], 'z': [1],
    'u': [1], 'v': [1], 'w': [0]
}]

layout = {
    'scene': {
      'camera': {
        'eye': {'x': -0.76, 'y': 1.8, 'z': 0.92}
      }
    }
}

fig = {"data": data, "layout": layout}
iplot(fig, filename='cone-basic', validate=False)

```

**Comment by jonmmease** [comment]
@jzthree Can you try this out with the new version 3.0.1?

**Comment by jzthree** [comment]
Thanks. Now using `plotly.offline.plot` to save to a html file works perfectly, but `plotly.offline.iplot` still behaves like the above image I had.

**Comment by jonmmease** [comment]
Hi @jzthree , could you try the `plotly.offline.iplot` approach again with version 3.1.0?

**Comment by jzthree** [comment]
Thanks for the quick update!  I still have the same problem with 3.1.0 though...
![image](https://user-images.githubusercontent.com/8333155/43030698-04aa33fe-8c62-11e8-9e98-f100f40cff88.png)

**Comment by empet** [comment]
@jzthree  Any Plotly 3d figure that hasn't defined `xaxis`, `yaxis`, `zaxis` in the `scene` dict is plotted as a 2d figure. Try  a `scatter3d`, a `surface` or a `mesh3d` type trace, without setting these keys in the layout `scene` and you'll see again 2d plots.

**Comment by jzthree** [comment]
Oh I just tried jupyter notebook and that works
![image](https://user-images.githubusercontent.com/8333155/43037469-36a5ca20-8cdb-11e8-8aa3-75a90c358a03.png)

The plot I showed above was nteract. I also used Atom+Hydrogen and it had the same problem, so I never thought it would be interface-dependent- sorry. I don't have enough expertise to tell whether this shoud be an issue for nteract @rgbkrk. Shoud we close this issue?
